metadata/rest/instrument_queries/query_base.py

```diff
@@ -9,7 +9,7 @@ class QueryBase(object):
     def format_instrument_block(instrument_entry):
         """Construct a dictionary from a given proposal instance in the metadata stack."""
         _ie = instrument_entry
-        name_components = re.search(r'(.+):\s*(.+)', str(_ie.name))
+        name_components = re.search(r'(.+?):\s*(.+)', str(_ie.name))
```

Comment:
This is also the EUS issue and should be removed... We have groups we should use them